block_carousel.php

```diff
@@ -126,7 +126,7 @@ public function get_content() {
         if ($slides > 1) {
             $firstslide = reset($data);
             $height = $firstslide['heightres'];
-            if ($height === 0) {
+            if ($height === 0 || $height === null) {
```

Comment:
Would it be simpler to use an empty check. I assume that height should only ever be a number or null, but an empty check might  catch cases where height is defined incorrectly as a string and should be regarded as 0 or null as well. 

Would also make reading and confirming the check it's doing a bit more straightforward.